db/migrate/20150224031239_add_email_settings_to_stages.rb

```diff
@@ -1,7 +1,7 @@
 class AddEmailSettingsToStages < ActiveRecord::Migration
   def change
     add_column :stages, :email_committers_on_automated_deploy_failure, :boolean, default: false, null: false
-    add_column :stages, :static_emails_on_automated_deploy_failure, :string
+    add_column :stages, :static_emails_on_automated_deploy_failure, :string, limit: 255
```

Comment:
I guess its conceivable for this not to be enough? Although maybe by that point its better to setup a mailing list.